content/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-session/index.en.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "ETC Core Devs Call 15: ECIP-1049 Keccak Breakout Session"
+date: 2020-10-02
+author: r0n1n
+tags: ["announcement", "development", "education", "teams", "hardfork", "media"]
+---
+
+![ETC Core Devs Call 15 - ECIP-1049 Keccak Breakout Session](./ethereum_classic_ecip_wallpaper.png)
```

Comment:
Please remove line 8, "ETC Core Devs Call." This was not such a meeting.

Reply:
This does not appear to be valid criticism of the PR. Another manufactured talking point by this participant.

Source: https://ethereumclassic.org/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-call